extensions/amp-story/1.0/amp-story-page-attachment.js

```diff
@@ -291,11 +291,10 @@ export class AmpStoryPageAttachment extends DraggableDrawer {
       // amp-story-page-attachment.css). The navigation itself will take some
       // time, depending on the target and network conditions.
       this.win.setTimeout(() => {
-        const navigationService = Services.navigationForDoc(this.getAmpDoc());
-        navigationService.navigateTo(
-          this.win,
-          this.element.getAttribute('href')
-        );
+        this.element.parentElement
+          .querySelector('.i-amphtml-page-attachment-host')
+          .shadowRoot.querySelector('a')
```

Comment:
1. What if `shadowRoot` is undefined because the browser doesn't support shadow DOM
2. Re: plz never rely on tag names in either CSS or JS. Always a class or something more reliable
3. Not your doing, but can you rename the `page-attachment-host`? It's super confusing, it's not the page attachment but the opening UI

Reply:
1. Taked offline, no need to change anything.
2. I'll use `a.i-amphtml-page-open-attachment` since that's more accurate. Using the `a` in the selector ensures it's an anchor tag that can be clicked.
3. `...page-open-attachment` seems to be the way the open attachment UI is named. I'll change it.